Test failure on Node 4 "node: bad option: --harmony_collections"

```
$ npm test

> serializable@1.0.0 test /Users/dan/stuff/serializable
> grunt test

Running "coffee:glob_to_multiple" (coffee) task
File lib/serializable.js created.

Running "coffeelint:src" (coffeelint) task
>> 1 file lint free.

Running "coffeelint:test" (coffeelint) task
>> 2 files lint free.

Running "coffeelint:gruntfile" (coffeelint) task
>> 1 file lint free.

Running "shell:test" (shell) task
node: bad option: --harmony_collections
Warning: Command failed: /bin/sh -c node --harmony_collections node_modules/.bin/jasmine-focused --coffee --captureExceptions spec
node: bad option: --harmony_collections
 Use --force to continue.

Aborted due to warnings.
npm ERR! Test failed.  See above for more details.
```

--- a/Gruntfile.coffee
+++ b/Gruntfile.coffee
@@ -25,7 +25,7 @@ module.exports = (grunt) ->
 
     shell:
       test:
-        command: 'node --harmony_collections node_modules/.bin/jasmine-focused --coffee --captureExceptions spec'
+        command: 'node node_modules/.bin/jasmine-focused --coffee --captureExceptions spec'
         options:
           stdout: true
           stderr: true

